app/models/setting.rb

```diff
@@ -83,4 +84,38 @@ def self.invoice_reminder_in_days
   def self.wishlist_size
     Setting.find_by(code: :wishlist_size).value.to_i
   end
+
+  def self.wishlist_supported_domain_extensions
```

Comment:
This method seems quite uncertain (too many `if`s). In what case the setting is absent? It seems `return...` is the mainline logic here, so how about moving it to the end of the method and `return []` before it? (Guard Clause https://refactoring.com/catalog/replaceNestedConditionalWithGuardClauses.html)?